lightning/src/util/config.rs

```diff
@@ -269,6 +269,13 @@ pub struct ChannelConfig {
 	/// [`Normal`]: crate::chain::chaininterface::ConfirmationTarget::Normal
 	/// [`Background`]: crate::chain::chaininterface::ConfirmationTarget::Background
 	pub force_close_avoidance_max_fee_satoshis: u64,
+	/// Sets how many the percent of the channel value the holder_max_htlc_value_in_flight_msat
```

Comment:
Need *way* more docs here - without knowing anything about the lightning protocol, a user reading this documentation should be able to identify what the imlications of setting this value higher or lower will be.

It should probably link to `ChannelConfig::cltv_expiry_delta` and `ChannelHandshakeConfig::our_to_self_delay`, noting that this limits the HTLC exposure for inbound HTLCs only, It may also note that there is currently no ability to limit the outbound HTLC exposure.

Reply:
Ok thanks, will update!